package.json

```diff
@@ -17,7 +17,9 @@
         "preset": "jest-expo"
     },
     "dependencies": {
+        "@expo/ngrok": "^4.1.3",
         "@expo/vector-icons": "^14.0.2",
+        "@react-native-community/datetimepicker": "^8.2.0",
```

Comment:
I know for a fact that we use this dependency, but I think some of the other dependencies might be from the previous date-time picker you were using. Please uninstall the dependencies that we no longer need.